screwdriver.yaml

```diff
@@ -11,7 +11,8 @@ jobs:
             - install: go mod download
             - vet: go vet ./...
```

Comment:
If we use `make test`, `vet` and `gofmt` steps are duplicated.

Reply:
Removed `go vet`.
`gofmt` is left because the options used by Makefile and screwdriver.yaml are different.